components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -164,8 +165,6 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('sdate', startdate)
 	Variables.varDefine('edate', enddate)
 
-	Variables.varDefine('game', GameLookup.getName({args.game}))
```

Comment:
we sure this wiki var isn't used anymore? (aoe still on match1, so there it might atill be used; i am on phone and checking it on phone is shit)